e2e-tests/common/utils.resource

```diff
@@ -0,0 +1,123 @@
+*** Settings ***
+Documentation       authd test resources
+
+Resource            kvm.resource
+Library             ${CURDIR}/StringUtils.py    AS    StringUtils
+Library             Process
+
+
+*** Variables ***
+${IMAGES_DIR}    ${CURDIR}/images
+${SCRIPTS_DIR}    ${CURDIR}/scripts
+
+
+*** Keywords ***
+Log In
+    PlatformVideoInput.Match Text    Not listed    120
+    PlatformHid.Keys Combo    Return
+    BuiltIn.Sleep    2
+    PlatformHid.Type String    ubuntu
+    BuiltIn.Sleep    2
+    PlatformHid.Keys Combo    Return
+    BuiltIn.Sleep    2
+    PlatformVideoInput.Match    ${IMAGES_DIR}/circle-of-friends.png    60
+
+
+Log Out
+    Run Command    gnome-session-quit --logout --no-prompt
+
+
+Open GNOME Terminal
+    Start Application    /usr/bin/gnome-terminal
+    PlatformVideoInput.Match Text    @ubuntu:~$    15
+    PlatformHid.Keys Combo    Super_L    Up
```

Comment:
Add a comment to clarify that we're maximizing it.

I know I suggested this, but on a 2nd though I think it would be actually better to instead make it full-screen by using `F11`.

So we are sure that no other terminal window is bothering us, but this also implies waiting for the "Window title" instead that for the content of the window (as the content may be show by another terminal anyways).

Reply:
I'm not sure I like the full-screen approach, and it shouldn't make a difference compared to maximizing the terminal window, right?

Reply:
It does, because there's no other text involved a part the one in the terminal itself, and there's no risk that something in the panel or in some popping-up application may break us